static/js/settings_bots.js

```diff
@@ -86,13 +86,22 @@ exports.set_up = function () {
     $('#payload_url_inputbox').hide();
     $('#create_payload_url').val('');
     $('#service_name_list').hide();
-    page_params.realm_embedded_bots.forEach(function (bot_name) {
+    $('#config_inputbox').hide();
+    page_params.realm_embedded_bots.forEach(function (bot) {
         $('#select_service_name').append($('<option>', {
-            value: bot_name,
-            text: bot_name,
+            value: bot.name,
+            text: bot.name,
         }));
+        Object.keys(bot.config).forEach(function (key) {
```

Comment:
You could say `_.each(bot.config, function (value, key) {' here too.

Reply:
That's neat.